fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Oath Inc.
+// Copyright 2019 Verizon Media Group.
```

Comment:
Sadly we can't use Verizon Media Group yet as copyright.  We're still legally Oath.